src/gdb_packet.c

```diff
@@ -41,14 +41,23 @@ typedef enum packet_state {
 
 static bool noackmode = false;
 
+#ifdef EXTERNAL_PACKET_BUFFER
+extern gdb_packet_s *gdb_full_packet_buffer(void);
+#else
 /* This has to be aligned so the remote protocol can re-use it without causing Problems */
 static gdb_packet_s BMD_ALIGN_DEF(8) packet_buffer;
 
+static gdb_packet_s *gdb_full_packet_buffer(void)
```

Comment:
This function could do with being marked `inline` to strongly suggest to the compiler that it can, well, inline it.

Reply:
Done